packages/apputils/src/mainareawidget.ts

```diff
@@ -67,7 +67,6 @@ export class MainAreaWidget<T extends Widget = Widget>
     if (!content.id) {
       content.id = DOMUtils.createDomID();
     }
-    content.node.tabIndex = 0;
```

Comment:
@marthacryan could you take a look at this change?

After playing for some time with the JupyterLab UI, I couldn't find any good reason to give the main area widget a tab index. Every main area widget that I could find has some element within it that can receive focus, so there's no reason that I can think of to make the container receive focus. I thought maybe it had to do with widgets that only have content and no editing functions (like contextual help). I thought maybe putting a tab index on the container would help the user to use page up and page down keyboard keys to scroll the content, but in fact (on my Mac at least), when the container element has focus I cannot use the page up and page down keys to scroll.

Reply:
I wonder why the visual tests fail (and updates cannot be generated https://github.com/jupyterlab/jupyterlab/actions/runs/3503291407). Since all other changes are CSS-only, my suspicion is that galata may depend on `MainAreaWidget` being focusable. This is usually required in order to receive keyboard events. Could we try with

```
content.node.tabIndex = -1;
```

?

Reply:
@gabalafou I opened https://github.com/gabalafou/jupyterlab/pull/12. In particular tests hang on:

https://github.com/jupyterlab/jupyterlab/blob/72f89c9e7015dc3a79c6b5e2d6e2e479d9a8b926/galata/src/helpers/activity.ts#L91-L97

and this is visible in the UI as the tabs do not get the blue highlight the way they used to:

![focus-problem](https://user-images.githubusercontent.com/5832902/204394703-92c91f6c-3fb8-4cdf-b258-f4ad6d752f68.gif)

Reply:
Oh sorry thought I responded here! If you can't find a good reason to set that tabindex to 0 that's totally fine to remove. I went through and tried to remove a bunch of tabindex being set to -1 and I may have been setting them to 0 for no good reason. Thanks for working on this!